### ISSUE TITLE
Change the inflation rate to target % of atoms staked

### ISSUE THREAD

**Issue by jaekwon** [cosmos/cosmos#54]
e.g. 80%.

The original 20% may be a bit too high esp considering the unbonding period.

**Comment by LeeBlues** [comment]
I definitely agree with you. 20% seems high.

**Comment by LeeBlues** [comment]
I have been thinking about it. What about just apply DASH's inflation rate? 7% I guess. They have managed great reputation and all coin holders are satisfactory about this rate.

**Comment by ashhan8904** [comment]
If the inflation rate is too low, there is no room left for validating node fee and governance proposal budget. It takes money for the validating nodes especially when they are professional ones. This is a serious issue you should keep in mind. The inflation rate is not only for liquidity supply but the whole system.

**Comment by jaekwon** [comment]
Lets start with 7% and target a bond-rate of 2/3. 

The inflation rate will adjust to incentivize exactly 2/3 of the atoms to be bonded.
That way, there will hopefully be a sufficient market for atoms, but the atom is designed such that it is not a good medium of exchange.

There are legal and security elements to this.

Ash, if the inflation rate is too low, then we can maybe add a feature where bonded validators can unbond their atoms at some steady rate... with 0 interest rates, everyone will have to eat into their atoms to pay for bills.  I don't think this is necessarily a bad thing if everyone else is doing it too.

**Comment by ebuchman** [comment]
Does dash still use mining?

**Comment by ashhan8904** [comment]
Yes. It does. It is X11 algo together with maternodes.





Best Regards,

Ash Han

_____________________________________________


Blockchain and DLT Evangelist

Blog <https://brunch.co.kr/@ashhan>

Linkedin <https://www.linkedin.com/in/ash-han-seunghwan-a08551124/>

2017-03-24 10:05 GMT+09:00 Ethan Buchman <notifications@github.com>:

**Comment by ashhan8904** [comment]
Yes. 7% might be too low especially when the incentive for hacking a validating node is 5%(I think it should be lower than or close to 1%, which is still big). For the validating node, it would be more lucrative to get whatever amounts of atoms delegated by all means possible and just hack itself. 

The bonding reward should cover every cost coming from keeping necessary nodes/servers/network safe, and satisfy an expected income for delegators and, in addition, after all that, it still has to be higher than the incentive for hackers (default 5% as of now).

**Comment by AFDudley** [comment]
What's the policy/process for changing these magic constants in production?

**Comment by jaekwon** [comment]
> 7% might be too low especially when the incentive for hacking a validating node is 5%(I think it should be lower than or close to 1%, which is still big).

What's the relation?

**Comment by jaekwon** [comment]
> The bonding reward should cover every cost coming from keeping necessary nodes/servers/network safe, and satisfy an expected income for delegators(because delegators also have to profit from their own token bonding reward) and, in addition, after all that, it should still be higher than the incentive for hackers (default 5% as of now).

The system should reward itself via transaction fees.  Lets just consider this inflation rate a regulatory mechanism and not think of it as a reward mechanism.

> For the validating node, it would be more lucrative to get whatever amounts of atoms delegated by all means possible and just hack itself.

I think the way to address this is to ensure that enough atoms are burned such that we don't incentivize such behavior.  It can be solved independently of the 7% inflation rate.

@AFDudley The process is via governance, submitting a proposal that changes the parameter.

**Comment by AFDudley** [comment]
When the testnet is deployed will various agents also be deployed to validate these constants? We should create some pool of testnet tokens to break these constants, maybe 20%? ( Look another magic value :)

**Comment by jaekwon** [comment]
agents, like AI agents?  We could model that offline.

**Comment by AFDudley** [comment]
Yes and yes. I think both _may_ be required. But surely an offline simulation should be done with various network parameters and agent strategies.

**Comment by winslyn** [comment]
FYI, the highest annual interest rate for US was 14% in 1981 (https://fred.stlouisfed.org/series/INTDSRUSM193N). And the highest annual inflation rate for the past 60 years in US was 12.4%(https://data.bls.gov/timeseries/CUUR0000SA0L1E?output_view=pct_12mths). And S&P 500's compound return over 87 years was 9.8%.

I would shoot for 14%.
